src/javascript/JContent.register.jsx

```diff
@@ -34,14 +35,40 @@ export default function () {
             mode: state.jcontent.mode,
             params: state.jcontent.params
         }));
+
+        const permissions = useNodeChecks({
+            path: `/sites/${site}`,
+            language: language
+        }, {
+            requiredSitePermission: [JContentConstants.accordionPermissions.pagesAccordionAccess, JContentConstants.accordionPermissions.contentFolderAccordionAccess, JContentConstants.accordionPermissions.mediaAccordionAccess, JContentConstants.accordionPermissions.additionalAccordionAccess, JContentConstants.accordionPermissions.formAccordionAccess]
+        });
+
+        if (permissions.loading) {
+            return 'Loading...';
+        }
+
+        let defaultMode = JContentConstants.mode.PAGES;
+
+        if (!permissions.node.site.pagesAccordionAccess && !permissions.node.site.contentFolderAccordionAccess && !permissions.node.site.mediaAccordionAccess && !permissions.node.site.formAccordionAccess && !permissions.node.site.additionalAccordionAccess) {
```

Comment:
Maybe cleaner and more easier to understand if we check for the granted access rather than the negation. I'm thinking of something like:

```
let defaultMode = '';
if (permissions.node.site.pagesAccordionAccess) {
    defaultMode = JContentConstants.mode.PAGES;
} else if (permissions.node.site.contentFolderAccordionAccess) {
    defaultMode = JContentConstants.mode.CONTENT_FOLDERS;
} else if () {
    ....
}
```

Reply:
Make sense to me (My bad making it more complicated then neccessary)